README.md

```diff
@@ -1254,3 +1254,177 @@ docker run --net=host --entrypoint python3 -v /scality/ssd01/s3/scality-utapi/co
   --debug               Enable debug level logging (default: False)
   --dry-run             Don't do any computation. Only validate and print the configuration. (default: False)
 ```
+
+
+# UtapiV2 Service level metrics sidecar
+
+REST API to provide service level reports for UtapiV2
+
+## Usage
+
+```shell
+docker run -d \
+  --network=host \
+  -e SIDECAR_API_KEY=dev_key_change_me \
+  -e SIDECAR_SCALE_FACTOR=1.4 \
+  -e SIDECAR_WARP10_NODE="md1-cluster1:4802@127.0.0.1:4802" \
+  scality/s3utils service-level-sidecar/index.js
+
+curl -X POST -H "Authorization: Bearer dev_key_change_me" localhost:24742/api/report | jq
```

Comment:
A  centos 7 vm I have has 7.29.0 which is a few versions short of when that flag was added. I think we should stick with `-H` for compat, and also because it tells you the header format even if you're not using curl. (one may not be familiar with oauth2 headers)